internal/gps/source_manager.go

```diff
@@ -213,9 +213,9 @@ func NewSourceManager(c SourceManagerConfig) (*SourceMgr, error) {
 			lasttime = nowtime
 		}
 
-		if _, ok := err.(interface {
+		if t, ok := err.(interface {
 			Temporary() bool
-		}); ok {
+		}); ok && t.Temporary() {
```

Comment:
@matjam is this right?

Reply:
I'm pretty sure the old code was wrong; it's not enough to assert on the type, you gotta call the method. https://godoc.org/net#Error